packages/sources/synthetix-debt-pool/src/endpoint/debt.ts

```diff
@@ -34,8 +49,22 @@ const getTotalDebtIssued = async (
           'DebtCache',
         )
         const debtCache = new ethers.Contract(debtCacheAddress, DEBT_CACHE_ABI, networkProvider)
-        const [debtIssued] = await debtCache.currentDebt()
-        return debtIssued
+        const [debtIssued] = await debtCache.currentDebt({ blockNumber })
+
+        const synthetixBridgeAddress = await getContractAddress(
+          networkProvider,
+          config.chains[network].chainAddressResolverAddress,
+          getSynthetixBridgeName(network, jobRunID),
+        )
+        const synthetixBridge = new ethers.Contract(
+          synthetixBridgeAddress,
+          SYNTHETIX_BRIDGE_ABI,
+          networkProvider,
+        )
+        const synthTransferReceived = await synthetixBridge.synthTransferReceived({ blockNumber })
+        const synthTransferSent = await synthetixBridge.synthTransferSent({ blockNumber })
+        const issuedSynths = debtIssued.add(synthTransferReceived.sub(synthTransferSent))
```

Comment:
hi,

I know you followed our instructions for the equation for the function, but it turns out that this equation was inverse of what it should be. 

```
const issuedSynths = debtIssued.add(synthTransferSent.sub(synthTransferReceived))
```

we will update the text of the SIP

apologies for the confusion